src/sentry/south_migrations/0293_auto__del_unique_release_project_id_version.py

```diff
@@ -0,0 +1,711 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Removing unique constraint on 'Release', fields ['project_id', 'version']
+        db.delete_unique(u'sentry_release', ['project_id', 'version'])
```

Comment:
Can we change this to a `DROP INDEX CONCURRENTLY` call?

The index should be named `sentry_release_project_id_7309d47e8f37e7ff_uniq`.